pages/basePage.ts

```diff
@@ -0,0 +1,15 @@
+import { Page } from "@playwright/test";
+
+export class BasePage {
+  readonly page: Page;
+  readonly maxUserNameLength = 14;
+  readonly minUserNameLength = 3;
+
+  constructor(page: Page) {
+    this.page = page;
+  }
+  
+  async goToPage(source: string) {
```

Comment:
don't forget, we are using typescript. 
It means we need to use types. Types should be defined at least for variables and for method outputs. 
Check your code and add everywhere types for variables(const, let) and methods ouput